### PR TITLE
Mobile mode: adjust classes and styles with recent app-cs changes #3890

### PR DESCRIPTION
**Pull request by ashklianko**
-Extracting mobile mode functionality to a separate classes so archive (and any other class) could reuse it
-Reusing parents handling of selection and highlighting change

### CODE REVIEW

File: modules/lib/src/main/resources/assets/js/app/browse/ContentBrowsePanel.ts [hunk 1]
```diff
@@ -68,6 +60,9 @@ export class ContentBrowsePanel
     protected initElements() {
         super.initElements();
 
+        this.getBrowseActions().getAction(ActionName.UNDO_PENDING_DELETE).setVisible(false);
```
Comment (alansemenov) [inline modules/lib/src/main/resources/assets/js/app/browse/ContentBrowsePanel.ts]: We're no longer using Undo Pending Delete. Shouldn't we get rid of this action altogether?

**Review by alansemenov — COMMENTED** [review]
Nice refactoring

File: modules/lib/src/main/resources/assets/js/app/browse/ContentBrowsePanel.ts (hunk 1 above)
Comment (ashklianko) [reply modules/lib/src/main/resources/assets/js/app/browse/ContentBrowsePanel.ts]: Will remove

Comment (ashklianko) [reply modules/lib/src/main/resources/assets/js/app/browse/ContentBrowsePanel.ts]: Done

**Review by alansemenov — APPROVED** [review]

**Pull request merged**